sources/packages/web/src/services/http/SupportingUserApi.ts

```diff
@@ -13,21 +13,16 @@ export class SupportingUserApi extends HttpBaseClient {
     supportingUserType: SupportingUserType,
     payload: ApplicationIdentifierApiInDTO,
   ): Promise<ApplicationApiOutDTO> {
+    let response;
     try {
-      const response = await this.apiClient.post(
+      response = await this.postCall(
         this.addClientRoot(`supporting-user/${supportingUserType}/application`),
         payload,
-        this.addAuthHeader(),
       );
-      return response.data;
```

Comment:
mm...what was the issue here?

Reply:
I changed it as a @andrewsignori-aot suggestion as the other use cases are using postCall and handleAPICustomError.